src/meltano/cli/remove.py

```diff
@@ -63,7 +72,7 @@ def removal_manager_status_update(removal_manager: PluginLocationRemoveManager)
     plugin_descriptor = removal_manager.plugin_descriptor
     location = removal_manager.location
     if removal_manager.plugin_error:
-        message = removal_manager.remove_message
+        message = removal_manager.message
```

Comment:
The `remove_message` attribute was set to `None` at initialization time, but never used again.